### PR TITLE
Task track1 layout

### PR DESCRIPTION
**Pull request by JamesCullen2**

### CODE REVIEW

File: includes/track1.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Track  1 Layout</title>
+    <style>
```
Comment (o-tahboub) [inline includes/track1.html]: Should be moved later on to css folder.

File: includes/track1.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        .container {
+            width: 800px;
+        }
+
+        svg {
+            border: 1px solid black;
+        }
+
+        rect {
+            fill:#000000;
```
Comment (o-tahboub) [inline includes/track1.html]: Space after colon.

File: includes/track1.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        }
+
+        .path {
+            fill:#ffffff;
```
Comment (o-tahboub) [inline includes/track1.html]: Space after colon.

File: includes/track1.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    </style>
+</head>
+<body>
+    <div class="container">
+        <button>Return</button>
+        <svg width="1000" height="800">
+            <!--Background-->
+            <rect x="0" y="0" width="1000" height="800" class="bg" />
+            <rect x="40" y="10" width="910" height="30" class="path" />
+            <rect x="0" y="10" width="40" height="30" class="path finish" />
```
Comment (o-tahboub) [inline includes/track1.html]: Path class is not required so remove.

File: includes/track1.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            <rect x="940" y="10" width="30" height="400" class="path" />
+            <rect x="20" y="400" width="950" height="30" class="path" />
+            <rect x="850" y="420" width="30" height="200" class="path" />
+            <rect x="760" y="600" width="120" height="30" class="path" />
+            <rect x="740" y="600" width="30" height="150" class="path" />
+            <rect x="20" y="400" width="30" height="380" class="path" />
+            <rect x="20" y="750" width="300" height="30" class="path" />
+            <rect x="280" y="725" width="360" height="30" class="path" />
+            <rect x="600" y="750" width="360" height="30" class="path" />
+            <rect x="960" y="750" width="40" height="30" class="path finish" />
```
Comment (o-tahboub) [inline includes/track1.html:23]: Path class is not required so remove.

File: includes/track1.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            left: 480px;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <button>Return</button>
+        <svg width="1000" height="800">
+            <!--Background-->
+            <rect x="0" y="0" width="1000" height="800" class="bg" />
```
Comment (o-tahboub) [inline includes/track1.html]: class bg does not exist in styles.
Also, if it is referring to background, change the name to be more semantic, bg is not very clear.

**Review by o-tahboub — COMMENTED** [review]
Reviewed, see comments.

File: includes/track1.html (hunk 1 above)
Comment (JamesCullen2) [reply includes/track1.html]: all done- this is now in .css folder

File: includes/track1.html (hunk 2 above)
Comment (JamesCullen2) [reply includes/track1.html]: done

File: includes/track1.html (hunk 3 above)
Comment (JamesCullen2) [reply includes/track1.html]: done

File: includes/track1.html (hunk 4 above)
Comment (JamesCullen2) [reply includes/track1.html]: omitted

File: includes/track1.html (hunk 5 above)
Comment (JamesCullen2) [reply includes/track1.html:23]: omitted

File: includes/track1.html (hunk 6 above)
Comment (JamesCullen2) [reply includes/track1.html]: good point, well made.

**Review by o-tahboub — APPROVED** [review]
Comments responded to, approved.

**Pull request merged**